### PR TITLE
🏇 Job Board: Adding data models

### LINKED ISSUE #30: 🏇 Data models for the Job Board app
Tasks:
- [x] Create the following data models :
       - Job_Type
       - Location
       - Years_Of_Experience
       - Preference
- [x] Create data migrations for relevant data models:
       - Job_Type
       - Location
       - Years_Of_Experience 
- [x] Create tests for the models.

The data models in the ERD:

### PR DESCRIPTION
**Pull request by DeanBiton**
- Added all the data models relevant to the Job Board app.
- Registered the data models to the admin site.
- Added data migrations to relevant data models.
- Added tests for the data models

close #30

### CODE REVIEW

**Review by paOmer — APPROVED** [review]
LGTM!

**Review by taljacob2 — APPROVED** [review]
LGTM!

File: job_board/tests.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+@pytest.mark.django_db
+class TestJobBoardModels:
+
+    def test_data_migration(self):
+        assert JobType.objects.filter(text='Web developer').exists()
+        assert Location.objects.filter(name='Haifa', latitude=32.804135934085565, longitude=34.98975968417104).exists()
+        assert YearsOfExperience.objects.filter(text='no experience').exists()
+
+    def test_data_creation(self):
```
Comment (Yarboa) [inline job_board/tests.py:16]: If it is test_data_creation,
consider using fixtures, which are holders for new tests data

And then assert the content against the fixtures data

File: job_board/tests.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+from job_board.models.location import Location
+from job_board.models.years_of_experience import YearsOfExperience
+from job_board.models.preference import Preference
+import pytest
+
+
+@pytest.mark.django_db
+class TestJobBoardModels:
+
+    def test_data_migration(self):
```
Comment (Yarboa) [inline job_board/tests.py:11]: You can skip this test
Data migration is checking all model APIs are working

**Review by Yarboa — COMMENTED** [review]
Nice Work
Please refer my comments

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**